src/text_translation/bhashini/remote/model.py

```diff
@@ -41,5 +50,23 @@ async def inference(self, request: ModelRequest):
 
         response = await self.context.client.post(url, headers=headers, data=payload)
         resp = await response.json()
-        print(resp)
-        return {"translated": resp["text"], "success": True}
+        return resp["text"]
+    
+    def apply_dictionary(self, text, dictionary):
+        words = text.split()
+        modified_words = {}
+        for i in range(len(words)):
+            word = words[i]
+            if word in dictionary:
+                modified_words[i] = word
+                words[i] = dictionary[word]
```

Comment:
replacing with correct translation here and passing a word in a different language to be translated?